microsetta_admin/templates/scan.html

```diff
@@ -247,16 +247,18 @@ <h4>Scan History</h4>
                                 <th class="bordered">Scan Timestamp</th>
                                 <th class="bordered">Sample Status</th>
                                 <th class="bordered">Technician Notes</th>
+                                <th class="bordered">Observations</th>
                             </tr>
                         </thead>
                         <tbody>
-                        {% for scan in scans_info %}
-                            <tr>
-                                <td class="bordered">{{ format_timestamp(scan['scan_timestamp']) }}</td>
-                                <td class="bordered">{{ scan['sample_status'] }}</td>
-                                <td class="bordered">{{ scan['technician_notes'] }}</td>
-                            </tr>
-                        {% endfor %}
+                            {% for scan in scans_info %}
+                                <tr>
+                                    <td class="bordered">{{ format_timestamp(scan['scan_timestamp']) }}</td>
+                                    <td class="bordered">{{ scan['sample_status'] }}</td>
+                                    <td class="bordered">{{ scan['technician_notes'] }}</td>
+                                    <td class="bordered">{{ scan['observations'] }}</td>
+                                </tr>
+                            {% endfor %}
```

Comment:
When a single scan has multiple observations associated, the UI displays it as multiple copies of the scan, rather than correctly grouping the observations into a single scan.

![Screenshot 2024-06-17 at 17-18-18 Microsetta Admin](https://github.com/biocore/microsetta-admin/assets/83246693/a4e77833-e487-4621-8fcb-b67bcc756c89)

Reply:
Fixed